tests/integration/views/test_workspace.py

```diff
@@ -544,7 +544,7 @@ def test_workspace_multiselect_add_files_all_valid(airlock_client, bll):
                 "test/path1.txt",
                 "test/path2.txt",
             ],
-            "next_url": workspace.get_url("test/path.txt"),
+            "next_url": workspace.get_url(UrlPath("test/path.txt")),
```

Comment:
adding UrlPath to all of these paths does add a lot of noise, but I think it would make it easier to do [this refactor](https://github.com/opensafely-core/airlock/issues/622) in future.

I also feel that - `Workspace.get_url()` asks for a type, the tests should provide that type (rather than a string that happens to work).